src/main/java/ca/corefacility/bioinformatics/irida/database/changesets/AutomatedAnalysisToTemplate.java

```diff
@@ -0,0 +1,168 @@
+package ca.corefacility.bioinformatics.irida.database.changesets;
+
+import ca.corefacility.bioinformatics.irida.config.data.IridaApiJdbcDataSourceConfig;
+import ca.corefacility.bioinformatics.irida.exceptions.IridaWorkflowNotFoundException;
+import ca.corefacility.bioinformatics.irida.model.workflow.IridaWorkflow;
+import ca.corefacility.bioinformatics.irida.model.workflow.analysis.type.BuiltInAnalysisTypes;
+import ca.corefacility.bioinformatics.irida.model.workflow.description.IridaWorkflowParameter;
+import ca.corefacility.bioinformatics.irida.service.workflow.IridaWorkflowsService;
+import liquibase.change.custom.CustomSqlChange;
+import liquibase.database.Database;
+import liquibase.exception.CustomChangeException;
+import liquibase.exception.SetupException;
+import liquibase.exception.ValidationErrors;
+import liquibase.resource.ResourceAccessor;
+import liquibase.statement.SqlStatement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import javax.sql.DataSource;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Liquibase update to convert the project settings for automated Assembly and SISTR checkboxes to analysis templates.
+ */
+public class AutomatedAnalysisToTemplate implements CustomSqlChange {
+
+	private static final Logger logger = LoggerFactory.getLogger(AutomatedAnalysisToTemplate.class);
+	private IridaWorkflowsService workflowsService;
+	private DataSource dataSource;
+
+	@Override
+	public SqlStatement[] generateStatements(Database database) throws CustomChangeException {
+
+		JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
+
+		IridaWorkflow assemblyWorkflow = null;
+		IridaWorkflow sistrWorkflow = null;
+
+		//get the workflow information from the service
+		try {
+			assemblyWorkflow = workflowsService.getDefaultWorkflowByType(BuiltInAnalysisTypes.ASSEMBLY_ANNOTATION);
+		} catch (IridaWorkflowNotFoundException e) {
+			logger.warn(
+					"Assembly workflow not found.  Automated assemblies will not be converted to analysis templates.");
+			//Note this will definitely happen in the galaxy CI tests as only SNVPhyl and a test workflow are configured.
+		}
+
+		//get the workflow information from the service
+		try {
+			sistrWorkflow = workflowsService.getDefaultWorkflowByType(BuiltInAnalysisTypes.SISTR_TYPING);
+		} catch (IridaWorkflowNotFoundException e) {
+			logger.warn("SISTR workflow not found.  Automated SISTR will not be converted to analysis templates.");
+			//Note this will definitely happen in the galaxy CI tests as only SNVPhyl and a test workflow are configured.
+		}
+
+		//update assemblies
+		if (assemblyWorkflow != null) {
```

Comment:
It's not really necessary, but you could move the below code inside the `try-catch` block and it saves you having to check for `null` here.

Reply:
Sure it cleans it up a bit.  Done in 0b67979ce1